templates/node/cloudevents/index.js

```diff
@@ -18,7 +18,7 @@ const { CloudEvent, HTTP } = require('cloudevents');
  * See: https://github.com/knative-sandbox/kn-plugin-func/blob/main/docs/guides/nodejs.md#the-context-object
  * @param {CloudEvent} event the CloudEvent
  */
-function handle(context, event) {
+const handle = async (context, event) => {
```

Comment:
Why not just add the `async` keyword as in `async function handle()`?

Reply:
No particular reason, I just like arrow functions.

Reply:
Also `function` binds `this` to something. Do we want that something expose?

Reply:
I am not node expert, but I tend to agree with Lance, it is more obvious for me that way. But again, I am not expert in this area, so I am fine with any approach.

Reply:
@lance are we OK with arrow functions or shall I revert to `function`.